.github/workflows/codeql.yml

```diff
@@ -0,0 +1,39 @@
+name: "CodeQL"
+
+on:
+  pull_request:
+    branches: ["main"]
```

Comment:
Seems it's not happy and wants this too:

```
  push:
    branches: ["main"]
```

Reply:
Give a finger and they take the arm :roll_eyes: